bundles/binding/org.openhab.binding.dsmr/src/main/java/org/openhab/binding/dsmr/internal/DSMRBinding.java

```diff
@@ -194,6 +197,15 @@ public void updated(Dictionary<String, ?> config) throws ConfigurationException
                 logger.warn("dsmr:port setting is empty");
             }
 
+            // Read port settings
+            String portSettingsString = (String) config.get("portsettings");
+            logger.debug("dsmr:portsettings=" + portSettingsString);
+            if (StringUtils.isNotBlank(portSettingsString)) {
+                fixedPortSettings = DSMRPortSettings.getPortSettingsFromString(portSettingsString);
+            } else {
+                logger.info("dsmr:portsettings setting is empty. Ignored");
```

Comment:
I didn't incorporate this. If the dsmr:port is empty the binding probably won't work. Hoewever dsmr:portsettings is not mandatory and will not affect the correct working.